src/main/java/it/gov/pagopa/payhub/activities/dto/IngestionFlowFileDTO.java

```diff
@@ -32,4 +34,10 @@ public class IngestionFlowFileDTO implements Serializable {
     private Long pdfGenerated;
     private String codRequestToken;
     private String codError;
+    private String pspIdentifier;
+    private Date flowDateTime;
```

Comment:
let's use new date/time API

are these columns on the table that Matteucci is creating?
```suggestion
    private LocalDateTime flowDateTime;
```